source/module_hsolver/kernels/test/perf_math_kernel.cpp

```diff
@@ -0,0 +1,108 @@
+#include "module_base/blas_connector.h"
+#include "module_base/constants.h"
+#include "module_hsolver/kernels/math_kernel_op.h"
+#include "module_psi/kernels/memory_op.h"
+
+#include <complex>
+#include <benchmark/benchmark.h>
+#include <iostream>
+
+class PerfModuleHsolverMathKernel : public benchmark::Fixture {
+    public:
+
+    // DEVICE SYMBOL
+    const psi::DEVICE_CPU* cpu_ctx = {};
+    const psi::DEVICE_GPU* gpu_ctx = {};
+
+    int dim_vector = 1;
+
+    std::complex<double>* test_zvector_a,* test_zvector_b,* result_zvector;
+
+    double* test_dvector_a;
+
+    double constant = 1.24;
+
+    void SetUp(const benchmark::State& state){
+        dim_vector = state.range(0) * state.range(0) * 50000 + 1000; // Generate vectors with different diminsions
```

Comment:
How can user test the dim_vector of {1, 10, 100, 1000, 1e4, 1e5, 1e6, 1e7 ...} or other special lengths?

Reply:
I expect to output the time of different 'dim_vector' separately.

Reply:
The exact dim of vector can only be set in code. So I will adjust it to a appropriate dim sequence later.
`state.range(0)` is an argument passed by google benchmark itself.